Makefile

```diff
@@ -1,16 +1,31 @@
-.PHONY: clean install ci-install test bundle
+.PHONY: clean install ci-install test test-integ test-docker bundle
 
 clean:
 	@rm -rf vendor composer.lock sendgrid-php.zip
 
 install: clean
+ifdef GIT_HUB_TOKEN
+	composer config -g github-oauth.github.com $(GIT_HUB_TOKEN)
+endif
+
 	composer install
 
+ifeq ($(dependencies), lowest)
+	composer update --prefer-lowest --prefer-stable -n
+endif
+
 ci-install: clean
 	composer install --no-dev
 
-test: install
+test:
 	vendor/bin/phpunit test/unit --filter test*
 
+test-integ: test
+	vendor/bin/phpunit test --filter test*
+
+test-docker:
```

Comment:
This command is erroring for me, with "Service 'helper-runner' failed to build: invalid reference format"

Reply:
Yeah, it's a _bug_ with docker compose. I'll update with a workaround.

Reply:
Both PRs updated.